requests_toolbelt/adapters/appengine.py

```diff
@@ -27,6 +31,7 @@ class AppEngineAdapter(adapters.HTTPAdapter):
         >>> from requests_toolbelt.adapters import appengine
         >>> s = requests.Session()
         >>> if using_appengine():
```

Comment:
It's not meant to be literal code. It's pseudo-code and many people abhor monkey-patching libraries @mikelambert so it is an example for them as to how to do this.